framework/core/allocated.h

```diff
@@ -28,10 +28,28 @@ class Device;
 
 namespace allocated
 {
+/**
+ * @brief Retrieves a reference to the VMA allocator singleton.  It will hold a null value except between
+ * calls to `init` and `shutdown`.  Otherwise it contains the opaque handle to the VMA allocator.
```

Comment:
Ahm... other way round?
Will contain the opaque handle between calls to 'init' and 'shutdown'. Otherwise a null value.